### DISCUSSION TITLE
if I use gpu to train,How can I best set these three environment variables?

### DISCUSSION THREAD

**Discussion by shenghansen** [deepmodeling/deepmd-kit#1512] [Q&A]
when i use gpu to train ,I met the warning  below，How can I best set these three environment variables? 

WARNING:root:To get the best performance, it is recommended to adjust the number of threads by setting the environment variables OMP_NUM_THREADS, TF_INTRA_OP_PARALLELISM_THREADS, and TF_INTER_OP_PARALLELISM_THREADS.

**Comment by shenghansen** [comment]
I know this three environment variables mean,but i don't know how to set them to get the best performance if i use gpu ,should I set OMP_NUM_THREADS=4 if i have 4 gpus,then how to set TF_INTRA_OP_PARALLELISM_THREADS, and TF_INTER_OP_PARALLELISM_THREADS.I really need your help,please.

**Comment by njzjz** [comment]
It's unclear unless you compare on your machine.
